spec/datastreams/content_metadata_ds_spec.rb

```diff
@@ -248,14 +248,14 @@
   end
 
   describe '#contentType=' do
-    let(:contentMetadata) { described_class.new }
+    let(:content_metadata) { described_class.new }
```

Comment:
didn't touch this code, only to make rubocop happy

Reply:
Lets take care of that separately then: https://github.com/sul-dlss/dor-services/pull/714

Reply:
Hmm... I merged that PR, and then rebase on this branch, but now I get various other rubocop violations in other parts of the code.

Reply:
You have to rebase and then fix them here or this will cause those errors on master

Reply:
Backed this change out of this PR and created a new PR for this and some other rubocop fixes I saw in master branch: #715

Reply:
I'm confused.  I've not updated my local rubocop gem for this project but it looks like I'm now using the same gem as travis, but the errors I am getting (none now) do not align with theirs.

Reply:
Ah, ok, I needed to update other rubocop related gems.  Now I get the same errors as travis...back to another PR i guess.

Reply:
Ok, new rubocop PR to fix failures on master branch, can rebase to them here too: #716